docs/src/components/E2EExample.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+import { Routes as BrowserTestRoutes } from '../../../e2e'
```

Comment:
```suggestion
import { routes as browserTestRoutes } from '../../../e2e'
```

nit: it's a just variables, so it can be camelcased